static/index.html

```diff
@@ -7,25 +7,29 @@
     <link href="out/styles/app.css" rel="stylesheet" />
     <link rel="apple-touch-icon" sizes="180x180" href="apple-touch-icon.png" />
     <link rel="icon" type="image/png" sizes="32x32" href="media/favicon.png" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css" />
```

Comment:
you should bundle them instead of linking a CDN.

Reply:
> you should bundle them instead of linking a CDN.

Makes sense, i bundled it